srcpkgs/python3-orocos-kdl/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'python3-orocos-kdl'
+pkgname=python3-orocos-kdl
+version=1.4.0
+revision=1
+wrksrc=orocos_kinematics_dynamics-${version}
+build_wrksrc=python_orocos_kdl
+build_style=cmake
+hostmakedepends="python3 python3-sip"
+makedepends="python3-sip-devel python3-devel orocos-kdl-devel eigen3.2"
+depends="python3 python3-sip"
+short_desc="Python3 Bindings for Orocos Kinematics and Dynamics C++ library"
+maintainer="Kyle Nusbaum <knusbaum+void@sdf.org>"
+license="LGPL-2.1-only"
+homepage="http://www.orocos.org/kdl"
+distfiles="https://github.com/orocos/orocos_kinematics_dynamics/archive/v${version}.tar.gz"
+checksum=05b93e759923684dc07433ccae1e476d158d89b3c2be5079c20062406da7b4dd
+
+configure_args="-DPYTHON_VERSION=3"
```

Comment:
add it directly below build_style=

Reply:
ok. I tried xlint and it didn't complain.